providers/ms365/resources/ms365.lr

```diff
@@ -1106,6 +1106,22 @@ microsoft.policies {
   authenticationMethodsPolicy() microsoft.policies.authenticationMethodsPolicy
   // Activity-based timeout policies
   activityBasedTimeoutPolicies() []microsoft.policies.activityBasedTimeoutPolicy
+  // Tenant-wide policy that controls whether external users can leave a tenant
+  externalIdentitiesPolicy() microsoft.policies.externalIdentitiesPolicy
+}
+
+// Tenant-wide policy that controls whether external users can leave a tenant via self-service controls
+private microsoft.policies.externalIdentitiesPolicy @defaults("displayName allowExternalIdentitiesToLeave") {
+  // Policy ID
+  id string
+  // Policy display name
+  displayName string
+  // Policy description
+  description string
+  // Defines whether external users can leave the guest tenant. If set to false, self-service controls are disabled, and the admin must manually remove the external user.
+  allowExternalIdentitiesToLeave bool
+  // Reserved for future use
```

Comment:
I do not understand this comment.
I see, that this field is populated: https://github.com/mondoohq/cnquery/pull/5744/files#diff-d145592a51f5f5351ce5cd309390592b27c3201504d24fbefdaf0a1986fa02a9R289

Should this have a comment like you set for `allowExternalIdentitiesToLeave`?